server/public/model/manifest.go

```diff
@@ -188,6 +188,9 @@ type Manifest struct {
 
 	// Plugins can store any kind of data in Props to allow other plugins to use it.
 	Props map[string]any `json:"props,omitempty" yaml:"props,omitempty"`
+
+	// To show the plugin in market place, if set to true then hide the plugin from market place plugin list
+	DisableInMarketPlace bool `json:"disable_in_marketplace,omitempty" yaml:"disable_in_marketplace,omitempty"`
```

Comment:
Let's be specific that this is to hide from the marketplace for prepackaged plugins: 

```suggestion
	// If set to true then skip listing the plugin in marketplace if it is prepackaged in the server
	SkipMarketplaceIfPrepackaged bool `json:"skip_marketplace_if_prepackaged,omitempty" yaml:"skip_marketplace_if_prepackaged,omitempty"`
```

Reply:
Okay